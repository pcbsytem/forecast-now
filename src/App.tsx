import React from "react";
import { Provider } from "react-redux";
import Weather from "./modules/weather/Weather";
import Store from "./store/Store";
import "./App.css";

function App() {
  return (
    <Provider store={Store}>
      <Weather />
    </Provider>
  );
}

export default App;
