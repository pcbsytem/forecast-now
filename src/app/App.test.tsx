import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

describe("App component", () => {
  test("Should render correctly", () => {
    expect(render(<App />)).toMatchSnapshot();
  });
});
