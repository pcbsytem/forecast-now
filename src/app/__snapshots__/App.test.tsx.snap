// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-hKMtZM hcKNel rs-flex-box-grid rs-flex-box-grid-top rs-flex-box-grid-center"
      >
        <h1
          class="sc-eCYdqJ ihCEgh"
        >
          Weather Forecast
        </h1>
        <p
          class="sc-jSMfEi gwwopn"
        >
          Search the weather forecast with your address.
        </p>
        <div
          class="sc-iqcoie locHMB rs-flex-box-grid rs-flex-box-grid-top rs-flex-box-grid-start"
        >
          <div
            class="sc-crXcEl hfqiqF rs-input-group rs-input-group-inside"
            width="100%"
          >
            <input
              class="rs-input"
              placeholder="Example: 1600 Pennsylvania Avenue NW, Washington"
              type="text"
              value=""
            />
            <button
              class="rs-input-group-addon rs-input-group-btn rs-btn rs-btn-default"
              type="button"
            >
              <svg
                aria-hidden="true"
                aria-label="search"
                class="rs-icon"
                data-category="action"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
              >
                <path
                  d="M6.5 1a5.5 5.5 0 110 11 5.5 5.5 0 110-11zm0 1a4.5 4.5 0 100 9 4.5 4.5 0 000-9z"
                />
                <path
                  d="M9.646 9.646a.5.5 0 01.638-.058l.069.058 4.5 4.5a.5.5 0 01-.638.765l-.069-.058-4.5-4.5a.5.5 0 010-.707z"
                />
              </svg>
              <span
                class="rs-ripple-pond"
              >
                <span
                  class="rs-ripple"
                />
              </span>
            </button>
          </div>
        </div>
        <div
          class="sc-gKXOVf bdZBgz"
        >
          <svg
            height="360px"
            width="360px"
          >
            cloudyday.svg
          </svg>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-hKMtZM hcKNel rs-flex-box-grid rs-flex-box-grid-top rs-flex-box-grid-center"
    >
      <h1
        class="sc-eCYdqJ ihCEgh"
      >
        Weather Forecast
      </h1>
      <p
        class="sc-jSMfEi gwwopn"
      >
        Search the weather forecast with your address.
      </p>
      <div
        class="sc-iqcoie locHMB rs-flex-box-grid rs-flex-box-grid-top rs-flex-box-grid-start"
      >
        <div
          class="sc-crXcEl hfqiqF rs-input-group rs-input-group-inside"
          width="100%"
        >
          <input
            class="rs-input"
            placeholder="Example: 1600 Pennsylvania Avenue NW, Washington"
            type="text"
            value=""
          />
          <button
            class="rs-input-group-addon rs-input-group-btn rs-btn rs-btn-default"
            type="button"
          >
            <svg
              aria-hidden="true"
              aria-label="search"
              class="rs-icon"
              data-category="action"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="0 0 16 16"
              width="1em"
            >
              <path
                d="M6.5 1a5.5 5.5 0 110 11 5.5 5.5 0 110-11zm0 1a4.5 4.5 0 100 9 4.5 4.5 0 000-9z"
              />
              <path
                d="M9.646 9.646a.5.5 0 01.638-.058l.069.058 4.5 4.5a.5.5 0 01-.638.765l-.069-.058-4.5-4.5a.5.5 0 010-.707z"
              />
            </svg>
            <span
              class="rs-ripple-pond"
            >
              <span
                class="rs-ripple"
              />
            </span>
          </button>
        </div>
      </div>
      <div
        class="sc-gKXOVf bdZBgz"
      >
        <svg
          height="360px"
          width="360px"
        >
          cloudyday.svg
        </svg>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
